jekyll/_cci2/bundling-config.md

```diff
@@ -0,0 +1,905 @@
+  - run: echo "And now I'm going to run the tests"
+  - run: make test
+```
+
+#### Enum Parameter
+{:.no_toc}
+
+Use the `enum` parameter to declare the target operating system for a binary.
+
+```yaml
+commands:
+  list-files:
+    parameters:
+      os: 
+        default: "linux"
+        description: The target Operating System for the heroku binary. Must be one of "linux", "darwin", "win32".
+        type: enum
+        enum: ["linux", "darwin", "win32"]
+```        
+
+The following `enum` type declaration is invalid because the default is not declared in the enum list.
+
+```yaml
+commands:
+  list-files:
+    parameters:      
+      os:
+        type: enum
+        default: "windows" #invalid declaration of default that does not appear in the comma-separated enum list
+        enum: ["darwin", "linux"]
+```        
+
+## Authoring and Using Jobs
+
+**Note:** Invoking jobs multiple times in a single workflow and parameters in jobs are available in configuration version 2.1 and later.
+
+Jobs are sets of steps and the environments they should be executed within.
+
+### Job Naming and Organization
+{:.no_toc}
+
+Jobs are defined in your build configuration. Job names are defined
+in a map under the `jobs` key in configuration.
+
+Like most elements, jobs can contain an optional but highly recommended `description`.
+
+A user must invoke jobs in the workflows stanza of `config.yml`, passing any necessary parameters as subkeys to the job. See the parameters documentation for more detailed information.
+
+Example of defining and invoking a parameterized job in a `config.yml`:
+
+```yaml
+version: 2.1
+
+jobs:
+  sayhello:
+    description: A job that does very little other than demonstrate what a parameterized job looks like
+    parameters:
+      saywhat:
+        description: "To whom shall we say hello?"
+        default: "World"
+        type: string
+    machine: true
+    steps:
+      - run: echo "Hello << parameters.saywhat >>"
+
+workflows:
+  build:
+    jobs:
+      - sayhello:
+          saywhat: Everyone
+```
+
+<!---
+### Jobs Defined in an orb
+
+If a job is declared inside an orb it can use commands in that orb or the global commands. We do not currently allow calling commands outside the scope of declaration of the job.
+
+**hello-orb**
+```yaml
+# partial yaml from hello-orb
+jobs:
+  sayhello:
+    parameters:
+      saywhat:
+        description: "To whom shall we say hello?"
+        default: "World"
+        type: string
+    machine: true
+    steps:
+      - say:
+          saywhat: "<< parameters.saywhat >>"
+commands:
+  saywhat:
+    parameters:
+      saywhat:
+        type: string
+    steps:
+      - run: echo "<< parameters.saywhat >>"
+```
+
+**Config leveraging hello-orb**
+```yaml
+# config.yml
+version: 2.1
+orbs:
+  hello-orb: somenamespace/hello-orb@volatile
+workflows:
+  build:
+    jobs:
+      - hello-orb/sayhello:
+          saywhat: Everyone
+```
+--->
+
+### Parameter Scope
```

Comment:
This should probably get moved up to the parameters section

Reply:
moved, thanks